client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -321,16 +321,8 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 			formattedHeader={
 				<FormattedHeader
 					id="domains-header"
-					align={
-						[ HUNDRED_YEAR_PLAN_FLOW, HUNDRED_YEAR_DOMAIN_FLOW ].includes( flow )
-							? 'center'
-							: 'left'
-					}
-					subHeaderAlign={
-						[ HUNDRED_YEAR_PLAN_FLOW, HUNDRED_YEAR_DOMAIN_FLOW ].includes( flow )
-							? 'center'
-							: undefined
-					}
+					align="center"
```

Comment:
Have we confirmed that the hundred year plan flow won't use this screen and this logic can be removed? Just curious.

Reply:
Yes, I confirmed the hundred-year requires center, and this screen also requires.